integrationTests/ts/package.json

```diff
@@ -6,11 +6,18 @@
   },
   "dependencies": {
     "graphql": "file:../graphql.tgz",
-    "typescript-4.1": "npm:typescript@4.1.x",
     "typescript-4.2": "npm:typescript@4.2.x",
     "typescript-4.3": "npm:typescript@4.3.x",
     "typescript-4.4": "npm:typescript@4.4.x",
     "typescript-4.5": "npm:typescript@4.5.x",
-    "typescript-4.6": "npm:typescript@4.6.x"
+    "typescript-4.6": "npm:typescript@4.6.x",
+    "typescript-4.7": "npm:typescript@4.7.x",
+    "typescript-4.8": "npm:typescript@4.8.x",
+    "typescript-4.9": "npm:typescript@4.9.x",
+    "typescript-5.0": "npm:typescript@5.0.x",
+    "typescript-5.1": "npm:typescript@5.1.x",
+    "typescript-5.2": "npm:typescript@5.2.x",
+    "typescript-5.3": "npm:typescript@5.3.x",
+    "typescript-5.4": "npm:typescript@5.4.x"
```

Comment:
How many versions *should* be supported here?

Reply:
I've chosen the same range as DefinitelyTyped for now: https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#support-window

We might choose to enforce more modern versions, though.

Reply:
this has been updated in main, maybe we can rebase this PR against main?